test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests_excludes.xml

```diff
@@ -43,14 +43,6 @@
 <exclude id="Excessive GC throws OOM" platform="Mode301" shouldFix="true"><reason>Metronome and Staccato do not use excessive GC</reason></exclude>
 <exclude id="Excessive GC appears in verbose log" platform="Mode301" shouldFix="true"><reason>Metronome and Staccato do not use excessive GC</reason></exclude>
 
-<!-- Metronome does not cause multiple verboseGC logging files -->
```

Comment:
These were added so that we don't run with Mode301 (Metronome). I suppose `playlist.xml ` now limites the different mode variants.